extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/usecase/UseCaseResource.java

```diff
@@ -48,22 +55,60 @@ public class UseCaseResource {
     private final SupportedPolicyApiService supportedPolicyApiService;
     /** Service for managing offerings. */
     private final OfferingService offeringService;
+    /** Service for managing data consumption */
+    private final ConsumptionService consumptionService;
 
     /**
      * Creates a new offer consisting of asset, policy definition and contract definition.
      *
-     * @param dto contains all required information for the offer.
-     * @return a 204 response, if creating the usecase was successful.
+     * @param dto {@link CreateOfferingDto} contains all required information for the offer.
+     * @return 204: if creating the usecase was successful.
      */
     @POST
-    @Path("contract-offer")
+    @Path("create-offer")
     @Consumes(MediaType.APPLICATION_JSON)
     @Operation(description = "Creates an offer")
     public Response createOfferEndpoint(CreateOfferingDto dto) {
         offeringService.create(dto);
         return Response.status(Response.Status.NO_CONTENT).build();
     }
 
+    /**
+     * Creates a new data consumption.
+     *
+     * @param dto {@link ConsumptionInputDto} contains all required information for a data consumption.
+     * @return 201: the consumption ID as {@link String}.
+     */
```

Comment:
we should remove the java doc in favor of the swagger annotations  which are the SSOT for our API documentation, as they are added to the OpenAPI YAML.

Regarding the Status 201, while it is possible to document it with the swagger annotations, it is quite verbose, and thus in question, whether it's worth it.

Regardless we should do one thing consistently. Either have it 201 + documented via the swagger annotations or just use 200 and skip the documentation.